beetsplug/bpd/__init__.py

```diff
@@ -640,8 +640,8 @@ class Command(object):
     """A command issued by the client for processing by the server.
     """
 
-    command_re = re.compile(br'^([^ \t]+)[ \t]*')
-    arg_re = re.compile(br'"((?:\\"|[^"])+)"|([^ \t"]+)')
+    command_re = re.compile(r'^([^ \t]+)[ \t]*')
+    arg_re = re.compile(r'"((?:\\"|[^"])+)"|([^ \t"]+)')
```

Comment:
Can we instead keep these as ordinary bytes literals and double-escape the backslashes (i.e., replace every `\` with `\\`)?